Documentation/operations/upgrade.rst

```diff
@@ -398,6 +398,7 @@ Added Metrics
 * ``cilium_identity_cache_timer_duration``
 * ``cilium_identity_cache_timer_trigger_latency``
 * ``cilium_identity_cache_timer_trigger_folds``
+* ``cilium__policy_incremental_update_duration``
```

Comment:
Is there an extra underscore there after `cilium`?

Reply:
sure is :blush: